wireless_modules/middle_module/MQTT_Client_class.py

```diff
@@ -0,0 +1,89 @@
+# Note: Can remove all the Print statements once all testing is successful
```

Comment:
By convention we use lowercase filenames, and there's also no need to have the `_class` suffix. `mqtt_client.py` would be good!